prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -0,0 +1,251 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Models;
+using Prime.Models.Api;
+using Prime.Services;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize(Policy = AuthConstants.USER_POLICY)]
```

Comment:
Rather than check for the feature flag in each endpoint, this can be updated to be
     [Authorize(Policy = AuthConstants.USER_POLICY, Roles = AuthConstants.FEATURE_SITE_REGISTRATION)]